Gemfile

```diff
@@ -3,6 +3,7 @@ git_source(:github){|repo| "https://github.com/#{repo}.git"}
 
 ruby "2.7.0"
 
+gem "bcrypt", "~> 3.1.13"
```

Comment:
gem này có tác dụng gì nhỉ

Reply:
has_secure_password là 1 cơ chế để bảo mật password; tạo ra trường password_digest trong db.
gem bcrypt cung cấp state-of-the-art hash function là brcypt để hash password, nâng cao tính bảo mật đối với tài khoản người dùng